src/components/editor/inputs/InputLang.jsx

```diff
@@ -51,6 +51,10 @@ const InputLang = (props) => {
   }
 
   const handleLangSubmit = (event) => {
+    if (radioButtonValue === "present" && lang === null) {
+      alert("Please select a valid language.")
+      return false
+    }
     close(event)
```

Comment:
Previously the user could click the "Select language" radio button but then not choose any language before clicking `Submit`.  This gives them an error if they try to do that.

Reply:
excellent.